src/components/AddBook.js

```diff
@@ -1,41 +1,67 @@
-import React from 'react';
+import React, { useRef } from 'react';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../redux/books/books';
 
-const AddBook = () => (
-  <section>
-    <h2>
-      ADD NEW BOOK
-    </h2>
-    <form>
-      <label htmlFor="title">
-        <input
-          type="text"
-          id="title"
-          name="title"
-          placeholder="Book title"
-        />
-      </label>
-      <label htmlFor="category">
+let id = 0;
+const AddBook = () => {
+  const dispatch = useDispatch();
+  let titleInput = useRef(null);
+  let categoryInput = useRef(null);
```

Comment:
Please ensure that the `New Book React Component` has a local state with `title` and `author` according to the project requirement.